backend/mega/mega.go

```diff
@@ -83,6 +83,14 @@ than permanently deleting them.  If you specify this then rclone will
 permanently delete objects instead.`,
 			Default:  false,
 			Advanced: true,
+		}, {
+			Name: "use_https",
+			Help: `Use HTTPS for transfers.
+
+HTTPS is not necesary since all data is already encrypted before being stored or transfered anyway.
+Enabling it will increase CPU usage and add network overhead.`,
```

Comment:
Perhaps you should write about this in the help too

> Some ISPs throttle HTTP connections, this causes transfers to become very slow.

Can you make sure the help text here is hard wrapped to 80 columns too please.

Reply:
I have updated it and reduced the line length.
https://github.com/rclone/rclone/blob/635400cfab18982c25a8dfad59e9c8711a150e46/backend/mega/mega.go#L88-L92

Reply:
Looks great :-)